### PR TITLE
doc for media

### PR DESCRIPTION
**Pull request by juchi**

### CODE REVIEW

File: en/developer_guide/media_gaufrette.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+By default, Open Orchestra stores the media on disk of the webserver of the application.
+If it is needed to change the location of the files, the configuration should be updated
+to change the adapter that Gaufrette must use for the file storage.
+
+All adapters that can be natively used are described in the `configuration documentation`_,
+along with the details of the configuration to set up.
+
+If another Gaufrette filesystem is used instead of the default one, it should be
+notified to Open Orchestra through the open_orchestra_media.filesystem configuration parameter,
```
Comment (itkg-bfouche) [inline en/developer_guide/media_gaufrette.rst]: markup : `open_orchestra_media.filesystem`

File: en/developer_guide/media_gaufrette.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+By default, Open Orchestra stores the media on disk of the webserver of the application.
+If it is needed to change the location of the files, the configuration should be updated
+to change the adapter that Gaufrette must use for the file storage.
+
+All adapters that can be natively used are described in the `configuration documentation`_,
+along with the details of the configuration to set up.
+
+If another Gaufrette filesystem is used instead of the default one, it should be
+notified to Open Orchestra through the ``open_orchestra_media.filesystem`` configuration parameter,
+which exepects the identifier of the new filesystem to use.
```
Comment (itkg-rboessel) [inline en/developer_guide/media_gaufrette.rst]: expects

**Comment by juchi** [comment]
ping @nicolasThal @itkg-bfouche @itkg-rboessel

File: en/developer_guide/media_gaufrette.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+to change the adapter that Gaufrette must use for the file storage.
+
+All adapters that can be natively used are described in the `configuration documentation`_,
+along with the details of the configuration to set up.
+
+If another Gaufrette filesystem is used instead of the default one, it should be
+notified to Open Orchestra through the ``open_orchestra_media.filesystem`` configuration parameter,
+which expects the identifier of the new filesystem to use.
+
+Upload and resizing of images
```
Comment (nicolasThal) [inline en/developer_guide/media_gaufrette.rst]: uploading and resizing or upload and resize

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**